### ISSUE TITLE
Outdated installation instructions in CS50 Manual

### ISSUE THREAD

**Issue by leviroth** [cs50/libcs50#57]
The instructions at https://manual.cs50.net/library/#installation still point to an older version of the CS50 Library. This has confused a fair number of people on the CS50x Slack. It would be nice to have the Manual updated, or at least to put up a warning so that people wanting to use the new `get_x` functions will find the right version.

**Comment by crossroads1112** [comment]
The instructions now seem to say "Coming Soon" so at the very least there isn't incorrect information out there. Updated installation instructions depend on #97

**Comment by dmalan** [comment]
This should be set, @leviroth!

https://cs50.readthedocs.io/library/c/
